lib/sli/nest-init.sli

```diff
@@ -732,9 +732,18 @@ def
    SeeAlso: Create, LayoutArray
 */
 
+/LayoutNetwork
+<< /deprecation_warning true >>
+Options
+
 /LayoutNetwork_l_a_dict
 {
   << >> begin
+  /LayoutNetwork /deprecation_warning GetOption true eq
+  {
+    (Sli function LayoutNetwork is deprecated in NEST 3.0.) M_DEPRECATED message
```

Comment:
"Sli" -> "SLI"

There are some more changes like this that I won't comment on. Please search and replace all of them.